README.md

```diff
@@ -41,30 +67,20 @@ How To (Plugin Developers)
 </dependency>
  ```
 
-**Or alternatively, with Gradle:**
+</p>
+</details>
 
- * Repository:
-```groovy
-repositories {
-    maven {
-        url 'https://papermc.io/repo/repository/maven-public/'
-    }
-}
-```
- * Artifact:
-```groovy
-dependencies {
-    compileOnly 'com.destroystokyo.paper:paper-api:1.16.1-R0.1-SNAPSHOT'
-}
-```
+ * See our API patches [here](Spigot-API-Patches)
+ * See upcoming, pending, and recently added API [here](https://github.com/PaperMC/Paper/projects/6)
+ * Paper API javadocs here: [papermc.io/javadocs](https://papermc.io/javadocs/)
 
-How To (Compiling Jar From Source)
+Compile From Source
 ------
-To compile Paper, you need JDK 8, maven, and an internet connection.
+To compile Paper, you need **JDK 8, maven, and an internet connection.**
```

Comment:
internet connection!?!?!? no shit sherlock when I'm reading this file I don't have that !!!!!!!1!!!1!!!!!!111!!! tahnk you
also you may want to change jdk8 to atleast jdk8, I am building everything I do with language level 8 but jdk11